src/app/auth/callback/route.js

```diff
@@ -0,0 +1,17 @@
+import { createRouteHandlerClient } from '@supabase/auth-helpers-nextjs';
```

Comment:
Do you think we should move this file into a `api` folder under `app` folder? 

Also noticing that this is a `.js` file. We should probably make this a `.ts` file

Reply:
I don't think I'm able to based since were using the app router
https://supabase.com/docs/guides/auth/auth-helpers/nextjs?language=ts
ctrl+f => route.js

Reply:
Should be able to 🤔 We def want all files to comply with being Typescript since the project is a using typescript.

I found this link: https://supabase.com/docs/guides/auth/auth-helpers/nextjs?language=ts#route-handlers

Will that help you?

Reply:
Whoops I should have clarified. I was able to covert into a TS file but I don't think I can change the file tree structure. Let me know if I'm missing something though!

https://supabase.com/docs/guides/auth/auth-helpers/nextjs?language=ts#managing-sign-in-with-code-exchange

Reply:
All g for now.